modules/installation-mirror-repository.adoc

```diff
@@ -183,4 +183,6 @@ that you selected, you must extract the installation program from the mirrored
 content.
 
 You must perform this step on a machine with an active Internet connection.
+
+If you are in a disconnected environment, use the `--image` flag as part of must-gather and point to the payload image . See the additional resources for more information about using must-gather.
```

Comment:
I would remove the sentence "See the additional resources for more information about using must-gather." There's a chance that this module could be used in an assembly that doesn't have that additional resources section.

Also, there is a space in front of the period that should be removed in the first sentence.